code/src/components/thoughts.css

```diff
@@ -0,0 +1,39 @@
+
+.thoughts__container {
+    background: #fff;
+    grid-column: span 3;
+    display: grid;
+    grid-template-columns: 1fr;
+}
```

Comment:
I now see that you made this container into a grid. Maybe it would be easier and less code if the grid for the whole page was specified in app__grid